spec/lib/linear_dispatcher_spec.rb

```diff
@@ -17,21 +17,21 @@
     end
 
     it "returns the first pending approval by position" do
-      proposal.approvals.create!(position: 6)
-      last_approval = proposal.approvals.create!(position: 5)
+      proposal.approvals.create!(position: 6, status: 'actionable')
+      last_approval = proposal.approvals.create!(position: 5, status: 'actionable')
```

Comment:
They shouldn't both be `actionable`, right? Also the description needs to be updated.